modules/components/nav.adoc

```diff
@@ -12,3 +12,4 @@
 ** xref:crypto.adoc[]
 ** xref:security.adoc[]
 ** xref:email.adoc[]
+** xref:selfservice.adoc[]
```

Comment:
Not sure self-service should be among "components". If so, should we perhaps mention it in `components/index.adoc`?

Reply:
I think we did not include it in the beginning as we thought it should be documented in wrensec-commons, but it makes sense to have it here as well.